gh-gl-sync/SpackCIBridge.py

```diff
@@ -415,9 +491,11 @@ def sync(self):
             # Find closed PRs that are currently synced.
             # These will be deleted from GitLab.
             closed_refspecs = self.get_prs_to_delete(open_prs["pr_strings"], synced_prs)
+            closed_refspecs = []
```

Comment:
Does this mean we're no longer deleting old branches from GitLab?

Reply:
Thank you 😅 that was for debugging purposes and should not remain.